properties.md

```diff
@@ -0,0 +1,140 @@
+# Explicit Bundle Properties
+
+A bundle `property` is a key/value pair that represents some aspect of a bundle in a consumer-agnostic way.
```

Comment:
I think this ship has sailed, but I would love to see how these definitions look if `bundles=[]properties` and we go turtles all the way down